types/ssri/ssri-tests.ts

```diff
@@ -178,3 +179,14 @@ ssri.integrityStream({
     integrity,
     algorithms: ["sha1", "sha256"],
 });
+
+// ssri.merge
+const data = fs.readFileSync('data.txt');
+const expectedIntegrity = ssri.parse(fs.readFileSync('integrity.txt', 'utf8'));
+const updatedIntegrity = ssri.fromData(data, { algorithms: ['sha512'] });
+expectedIntegrity.merge(updatedIntegrity); // $ExpectType void
+expectedIntegrity.merge(updatedIntegrity, {}); // $ExpectType void
+// $ExpectType void
+expectedIntegrity.merge(updatedIntegrity, {
+    strict: true,
+});
```

Comment:
This can be on a single line:
```suggestion
expectedIntegrity.merge(updatedIntegrity, { strict: true }); // $ExpectType void
```